internal/controller/ols_app_redis_assets.go

```diff
@@ -11,14 +17,52 @@ import (
 	olsv1alpha1 "github.com/openshift/lightspeed-operator/api/v1alpha1"
 )
 
+var DeploymentSelectorLabels = map[string]string{
```

Comment:
we can put "redis" into its name and convert it into a pacakge scoped function instead of a exported variable. 
```go
func generateRedisSelectorLabels() map[string]string {
	return map[string]string{
		"app.kubernetes.io/component":  "redis-server",
		"app.kubernetes.io/managed-by": "lightspeed-operator",
		"app.kubernetes.io/name":       "lightspeed-service-redis",
		"app.kubernetes.io/part-of":    "openshift-lightspeed",
	}
}

```

Reply:
Done, Changed as suggested.